styles/amplenote/catppuccin.user.css

```diff
@@ -0,0 +1,190 @@
+==/UserStyle== */
+
+@-moz-document domain("amplenote.com") {
+  :root:not(:has(body.theme-light)) {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+  :root:has(body.theme-light) {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+
+  #catppuccin(@lookup, @accent) {
+    @rosewater: @catppuccin[@@lookup][@rosewater];
+    @flamingo: @catppuccin[@@lookup][@flamingo];
+    @pink: @catppuccin[@@lookup][@pink];
+    @mauve: @catppuccin[@@lookup][@mauve];
+    @red: @catppuccin[@@lookup][@red];
+    @maroon: @catppuccin[@@lookup][@maroon];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    body.themeable {
+      --color-background-primary: @base;
+      --color-background-secondary: @mantle;
+      --color-background-tertiary: @mantle;
+
+      --color-background-popup: @mantle;
+      --color-background-header: @mantle;
+      --color-background-nav: @mantle;
+      --color-background-nav-active: @surface0;
+      --color-background-search-bar: @crust;
+      --color-background-nav-menu: @mantle;
+      --color-background-nav-menu-divider: @crust;
+      --color-background-popup-small: @surface0;
+      --color-background-reversed: @surface0;
+
+      --color-background-element-high-contrast: fade(@overlay2, 40%);
+      --color-background-element-medium-contrast: fade(@accent-color, 40%);
+      --color-background-element-low-contrast: @crust;
+
+      --color-background-note-menu-button: @surface0;
+      --color-background-banner: @surface0;
+      --color-background-nav-divider: @surface0;
+
+      --color-background-action-low-contrast: fade(@accent-color, 50%);
+      --color-background-action-high-contrast: @accent-color;
```

Comment:
```suggestion
      --color-background-action-low-contrast: fade(@accent-color, 50%);
      --color-background-action-medium-contrast: fade(@accent-color, 60%);
      --color-background-action-high-contrast: @accent-color;
```
<img width="431" alt="Screenshot 2024-06-30 at 9 25 55 AM" src="https://github.com/catppuccin/userstyles/assets/47499684/4dfac109-eccb-478c-b369-7d053ffaec39">

to

<img width="431" alt="Screenshot 2024-06-30 at 9 29 15 AM" src="https://github.com/catppuccin/userstyles/assets/47499684/be0177a8-c90e-4c49-b122-7d19b1e0734b">